packages/date-picker/test/keyboard-input.test.js

```diff
@@ -627,20 +627,6 @@ describe('keyboard', () => {
       expect(datepicker.value).to.equal('2000-01-01');
     });
 
-    it('should not be invalid on blur if valid date is entered', async () => {
-      await sendKeys({ type: '1/1/2000' });
-      await sendKeys({ press: 'Tab' });
-      expect(datepicker.invalid).not.to.be.true;
-    });
-
-    it('should validate on blur only once', async () => {
-      await sendKeys({ type: 'foo' });
-      const spy = sinon.spy(datepicker, 'validate');
-      await sendKeys({ press: 'Tab' });
-      expect(spy.callCount).to.equal(1);
```

Comment:
This check is covered by the following test:

https://github.com/vaadin/web-components/blob/59b9864003c9dec13d50c7687d48cde43daa32bd/packages/date-picker/test/validation.test.js#L66